src/executables/csql_launcher.c

```diff
@@ -311,6 +370,45 @@ main (int argc, char *argv[])
     {
       csql_arg.skip_column_names = false;
       csql_arg.plain_output = false;
+      csql_arg.query_output = false;
+      csql_arg.loaddb_output = false;
+    }
+
+  if (csql_arg.plain_output == true)
+    {
+      check_output++;
```

Comment:
What about change this variable as **num_output_styles**, for example, it doesn't matter but little confused to understand intention clearly.

Reply:
I will modify the variable name to **check_output_style** for increase understanding